flow-typed/npm/fetch-jsonp_vx.x.x.js

```diff
@@ -1,32 +1,18 @@
-// flow-typed signature: 40fcfe46cbedcd0342c9ee88f6c6aa21
-// flow-typed version: <<STUB>>/fetch-jsonp_v^1.0.2/flow_v0.40.0
-
-/**
- * This is an autogenerated libdef stub for:
- *
- *   'fetch-jsonp'
- *
- * Fill this stub out by replacing all the `any` types.
- *
- * Once filled out, we encourage you to share your work with the 
- * community by sending a pull request to: 
- * https://github.com/flowtype/flow-typed
- */
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 declare module 'fetch-jsonp' {
-  declare module.exports: any;
-}
+  declare module.exports: (url: string, options?: Options) => Promise<Response>;
 
-/**
- * We include stubs for each file inside this npm package in case you need to
- * require those files directly. Feel free to delete any files that aren't
- * needed.
- */
-declare module 'fetch-jsonp/build/fetch-jsonp' {
-  declare module.exports: any;
-}
+  declare type Options = {
+    timeout?: number,
+    jsonpCallback?: string,
+    jsonpCallbackFunction?: string,
+  };
 
-// Filename aliases
-declare module 'fetch-jsonp/build/fetch-jsonp.js' {
-  declare module.exports: $Exports<'fetch-jsonp/build/fetch-jsonp'>;
+  declare class Response {
+    json<T>(): Promise<T>,
+    ok: boolean,
+  }
```

Comment:
mostly stolen from https://github.com/camsong/fetch-jsonp/blob/master/index.d.ts

maybe I should write it somewhere :)